templates/chatbots/single_chatbot_home.html

```diff
@@ -219,11 +219,32 @@ <h3 class="font-bold text-lg">Link with {{ platform.label }}</h3>
 
     {% if perms.chat.view_chat %}
       <div role="tabpanel" class="tab-content" id="content-allsessions">
-        {% include "chatbots/components/session_table_actions.html" %}
-        <div id="sessions-table" data-url="{% url 'chatbots:sessions-list' request.team.slug experiment.id %}">
+        <div class="relative">
+          {% include "chatbots/components/session_table_actions.html" %}
+          <div class="absolute top-0 right-0 flex items-center gap-2">
+            <button class="btn btn-ghost btn-sm"
+                    hx-get="{% url 'chatbots:sessions-list' request.team.slug experiment.id %}"
+                    hx-target="#sessions-table"
+                    hx-swap="innerHTML"
+                    hx-indicator="#refresh-loader"
+                    title="Refresh sessions">
+              <i class="fa-solid fa-rotate"></i>
```

Comment:
add `htmx-hide` class to hide this icon during the request
```suggestion
              <i class="fa-solid fa-rotate htmx-hide"></i>
```